src/app/core/admin/admin-entity-details/admin-entity-field/admin-entity-field.component.html

```diff
@@ -79,7 +92,16 @@ <h2 mat-dialog-title i18n>Configure Field "{{ entitySchemaField.label }}"</h2>
             </mat-form-field>
 
             <mat-form-field>
-              <mat-label i18n>Type</mat-label>
+              <mat-label>
+                <span i18n>Type</span>
+                @if (data.overwriteLocally) {
+                  <fa-icon
```

Comment:
can we manage to put this as a tooltip on the whole field (using `matTooltipDisabled`) instead of the small icon? It felt easy to miss right now.